docs/checklist/resiliency.md

```diff
@@ -2,7 +2,7 @@
 title: Resiliency checklist
 description: Checklist that provides guidance for resiliency concerns during design.
 author: petertaylor9999
-ms.date: 03/24/2017
+ms.date: 03/24/201
```

Comment:
@jawn Minor thing, but can you revert this change? Then I'll merge. The other change looks good. Thanks!

Reply:
Done, updated the PR.